IdentityCore/src/broker_operation/request/interactive_token_request/MSIDBrokerOperationInteractiveTokenRequest.m

```diff
@@ -21,22 +21,28 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+#import <AuthenticationServices/ASAuthorizationOpenIDRequest.h>
```

Comment:
should this have ifdefs around to still compile with Xcode 10?

Reply:
added